src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,6 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
```

Comment:
@AndriiBardin I don't see this step `and test your Builder implementation here...`

Reply:
@boroda4436 fixed